src/patternfly/components/NotificationBadge/notification-badge.scss

```diff
@@ -53,3 +54,18 @@
     --pf-c-notification-badge--after--BorderColor: var(--pf-c-notification-badge--m-read--after--BorderColor);
   }
 }
+
+// When the notification badge in the masthead is focused, the pseudo element on the notification badge will have the same border color as the background color on the button.
+/* stylelint-disable */
+.pf-c-page__header-tools {
```

Comment:
should this also be in the page component's CSS?

Reply:
I wasn't sure what to call the variable in that case, because its overriding a variable in the notification badge component, or is it ok to use these variable names here @mcoker

Reply:
Oh if you added this in the page component, you would just overwrite `border-color` since you don't overwrite variables in other components. And I think it would be something like

```
--pf-c-page__header-tools__c-button--focus__c-notification-badge--m-unread--after--BorderColor
```

Reply:
thanks for clarifying got a little confused there, updated. I want to come back to this PR once we merge #1925